ircbot/plugins/currency.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from urllib.request import urlopen
+
+from ircbot import bot
+
+
+def get_currency(currency_a, currency_b):
+    endpoint = "http://api.fixer.io/latest?base={}".format(str(currency_a).upper())
+
+    response = urlopen(endpoint).read().decode("utf-8")
+    data = json.loads(response)
+
+    currency_ratio = data["rates"][str(currency_b).upper()]
+
+    return currency_ratio
+
+
+@bot.command('currency')
+def command(bot, channel, sender, args):
+    """Converts Currency. Usage: $currency 10 usd gbp"""
+    if len(args) < 3:
+        bot.message(channel, "This command requires 3 arguments.")
```

Comment:
Maybe here, re-iterate the usage of the command, just in case someone doesn't know